Resources/Prototypes/_Goobstation/Entities/Objects/Specific/syndicate.yml

```
@@ -82,3 +82,16 @@
   - type: Tag
     tags:
     - NukeOpsUplink
+
+- type: entity
+  parent: BaseUplinkRadio
+  id: BaseUplinkRadio225TCMedic
+  suffix: 225 TC, NukeOps medic
+  components:
+  - type: Store
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)